packages/browser/src/core/events/interfaces.ts

```diff
@@ -6,6 +7,10 @@ export type JSONValue = JSONPrimitive | JSONObject | JSONArray
 export type JSONObject = { [member: string]: JSONValue }
 export type JSONArray = Array<JSONValue>
 
+// TODO: the change is that context is no longer "undefined". "Callback" was defined twice in this codebase previously, with that being the only difference.
+// Are there methods where the context object for Callback is undefined? If so, which ones?
```

Comment:
🤔 can't think of any. Seems to be only used in the `dispatch` and ctx is always defined.

Reply:
Agreed, I can't think of when `ctx` would be undefined.